scripts/aem/grub.cfg

```diff
@@ -0,0 +1,37 @@
+set debug=linux,relocator,multiboot,slaunch
+serial --speed=115200 --word=8 --parity=no --stop=1
+terminal_input  serial
+terminal_output serial
+
+insmod part_msdos
+insmod ext2
+set root='hd0,msdos1'
+
+menuentry 'Minimal' {
+    echo    'Loading Linux ...'
+    linux /grub/bzImage placeholder initrd=initrd.img load_ramdisk=1 prompt_ramdisk=0 rw console=vga console=tty0 console=ttyS0,115200 earlyprintk=ttyS0,115200n8
+    echo    'Loading initial ramdisk ...'
+    initrd /grub/initramfs.cpio
+}
+
+menuentry 'Qubes, with Xen hypervisor' {
+    echo    'Loading Xen ...'
+    multiboot2      /grub/xen.gz placeholder console=tty0 console=ttyS0,115200 dom0_mem=min:1024M dom0_mem=max:4096M ucode=scan smt=off gnttab_max_frames=2048 gnttab_max_maptrack_frames=4096 loglvl=all guest_loglvl=all com1=115200,8n1 console=com1 ${xen_rm_opts}
+    echo    'Loading Linux ...'
+    module2 /grub/bzImage placeholder root=/dev/vda1 ro console=tty0 console=ttyS0,115200
+    echo    'Loading initial ramdisk ...'
+    module2 --nounzip   /grub/initramfs.cpio
+}
+
+menuentry 'Qubes, with Xen hypervisor and TrenchBoot' {
+    echo    'Enabling slaunch ...'
+    slaunch
+    slaunch_state
+    echo    'Loading Xen ...'
+    multiboot2      /grub/xen.gz placeholder console=tty0 console=ttyS0,115200 dom0_mem=min:1024M dom0_mem=max:4096M ucode=scan smt=off gnttab_max_frames=2048 gnttab_max_maptrack_frames=4096 loglvl=all guest_loglvl=all com1=115200,8n1 console=com1 ${xen_rm_opts}
+    echo    'Loading Linux ...'
+    module2 /grub/bzImage placeholder ro rhgb console=tty0 console=ttyS0,115200
+    echo    'Loading initial ramdisk ...'
+    module2 --nounzip   /initramfs.cpio
+    slaunch_state
```

Comment:
IIRC there is nothing that could impact any of the registers dumped by this command since previous invocation, so one of those can go.

Reply:
Removed the second one.